syntax = "proto3";

package oak;

message InitiateComputationRequest {
  bytes business_logic = 1;
  string expression = 2;
}

message InitiateComputationResponse {}

service OakServer {
  rpc InitiateComputation(InitiateComputationRequest) returns (InitiateComputationResponse);
}
